ggml/src/vulkan-shaders/vulkan-shaders-gen.cpp

```diff
@@ -481,6 +496,18 @@ int main(int argc, char** argv) {
         }
     }
 
+    if (argc <= 1 || std::string(argv[1]) == "--help") {
```

Comment:
```suggestion
    if (argc <= 1 || args.find("--help") != args.end()) {
```
This is nitpicking, but it's nice to be able to just put --help at the end of a longer command.